bitnami/nats/values.yaml

```diff
@@ -104,23 +104,32 @@ auth:
   usersCredentials: []
   noAuthUser: ""
 ## Cluster Configuration
+## ref: https://docs.nats.io/running-a-nats-service/configuration/clustering/cluster_config
 ##
 cluster:
+  ## @param cluster.name Cluster name
+  name: nats
   ## @param cluster.connectRetries Configure number of connect retries for implicit routes, otherwise leave blank
   ##
   connectRetries: ""
   ## Cluster Authentication
-  ## ref: https://github.com/nats-io/gnatsd#authentication
+  ## ref: https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro
   ## @param cluster.auth.enabled Switch to enable/disable cluster authentication
   ## @param cluster.auth.user Cluster authentication user
   ## @param cluster.auth.password Cluster authentication password
-  ## @param cluster.auth.token Cluster authentication token
```

Comment:
Removing this parameter since token-based authentication is only accepted for client connections but it's not supported for clustering